website/docs/components/button/components/playground.jsx

```diff
@@ -111,6 +111,7 @@ const Preview = (preview) => {
       loading={loading}
       disabled={disabled || loading}
       active={active}
+      ariaLabel="Text"
```

Comment:
We have not to popularize bad practice to put meaningless labels in our examples. Please  add prop to editable by user in playground